get-started-building-a-nuxt-3-blog-with-directus/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Build a blog with Nuxt 3 & Directus
+date: 2022-04-07
+author: Conner Bachmann
+image: ./build-a-blog-with-nuxt3-and-directus.webp
+category: integration
+---
+
+# Build a blog with Nuxt 3 & Directus
+
+> This guide shows you how to build a blog with Nuxt 3 and Directus step by step.
+
+[[toc]]
+
+### What is Nuxt 3?
+
+Nuxt 3 is an open source framework making web development simple and powerful. With Vue 3 and experience hybrid rendering, powerful data fetching and new features. [Visit the offical website](https://v3.nuxtjs.org).
+
+## Setup 🚀
+
+### 1. Create a new Nuxt 3 project
+
+```bash
+npx nuxi init my-blog
+
+cd my-blog/
+
+npm install
+```
+
+This created your Nuxt 3 application 💚.
+
+### 2. Create your Directus application [here](https://directus.cloud/login)
+
+### 3. Install the directus module & a CSS framework. We use TailwindCSS in this guide.
+
+```bash
+npm install nuxt-directus @nuxtjs/tailwindcss
+```
+
+### 4. Start the development server.
+
+```bash
+npm run dev
+```
+
+Your Nuxt 3 application is now running at <http://localhost:3000>.
+
+## Configuration ⚙️
+
+### 1. Update your `nuxt.config.ts`
+
+Change the URL to your Directus server in the configuration and add the modules we just installed. ✨
+
+```ts
+import { defineNuxtConfig } from "nuxt3";
+
+export default defineNuxtConfig({
+	modules: ["nuxt-directus", "@nuxtjs/tailwindcss"],
+	directus: {
+		url: "my-url",
+	},
+});
+```
+
+### Create a collection in the Directus admin dashboard 📄
+
+![image 1](1.webp)
+
+We call this collection `blog`, we can add an ID by selecting Type `Generated UUID`. Now, we check the box `date_created` to have a date for the blog post.
+
+### Create two new fields:
+
+![image 2](2.webp)
+
+Input field: `title` (type string **required**) - Title of the Blog article.
+Input field: `short_content` (type string **required**) - Short description of the article content.
+And a textarea field: `content` (type string **required**) - Article content.
+
+### Our blog data model should look like this:
+
+![image 3](3.webp)
+
+### Configure the directus permissions 🛑
+
+Add the permission to see all items to the public role in the just created `blog collection`.
+
+![image 4](4.webp)
+
+### Go to your collection and create example items.
+
+## You are ready to code! 💻
+
+### Open your Editor (In this Guide: Visual Studio Code) and go to the directory of the Nuxt project
+
+Delete the file `app.vue`, and create a new file `pages/index.vue`.
+
+**This is our new index page/home page.**
+
+Add the script to the page
+
+```ts
```

Comment:
We either have to use html/vue, or remove the `<script>` part for this to be a valid code block

```suggestion
```html
```